src/ansi-c/CMakeLists.txt

```diff
@@ -4,17 +4,10 @@ generic_flex(ansi_c)
 add_executable(converter library/converter.cpp)
 
 file(GLOB ansi_c_library_sources "library/*.c")
-set(converter_input_path "${CMAKE_CURRENT_SOURCE_DIR}/converter_input.txt")
-file(WRITE ${converter_input_path} "")
-foreach(source ${ansi_c_library_sources})
-    file(READ ${source} file_contents)
-    file(APPEND ${converter_input_path} "${file_contents}")
-endforeach()
 
 add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
-    COMMAND converter < ${converter_input_path} > "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
-    DEPENDS converter ${converter_input_path}
-)
+    COMMAND cat ${ansi_c_library_sources} | $<TARGET_FILE:converter> > "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
```

Comment:
Any particular reason for using cat here, rather than just stdin redirection from the file?

Reply:
@chrisr-diffblue Because it’s a list of files, not just one.

Reply:
Doh, of course.... :-)